src/Tizen.Multimedia.Remoting/MediaController/MediaControlCommand.cs

```diff
@@ -28,17 +29,23 @@ namespace Tizen.Multimedia.Remoting
     /// <since_tizen> 5 </since_tizen>
     public abstract class Command
     {
-        private string _requestId;
+        /// <summary>
+        /// The request id for each command.
+        /// </summary>
+        protected string _requestId;
```

Comment:
The protected variable is also be publicly exported. So It should be included in ACR scope. 
And the undersocre prefix should be used for private variables only.